CIScripts/Test/TestConfigurationUtils.ps1

```diff
@@ -469,7 +470,22 @@ function New-Container {
     if ($Name) { $Arguments += "--name", $Name }
     $Arguments += "--network", $NetworkName, $Image
 
-    $ContainerID = Invoke-Command -Session $Session { docker @Using:Arguments }
+    $Result = Invoke-NativeCommand -Session $Session -CaptureOutput -AllowNonZero { docker @Using:Arguments }
```

Comment:
Please use dependency injection to provide a string to docker executable. By default: `docker.exe` and in tests `TestDrive:\docker.ps1` for example

Reply:
I already use dependency injection - it's just on a bit different level. It's also more clean, because tested code doesn't need to have special features to make tests possible.

Adding/removing a single file on testbed also doesn't seem to matter. Our tests are already doing much more serious operations - installing msis, starting containers, etc. And TestDrive is another thing to care about when one tests the code without Pester.

Reply:
And TestDrive is available in session in which Pester is invoked. And this code is writing files in remote session.